flags.go

```diff
@@ -0,0 +1,144 @@
+package goforit
+
+import (
+	"encoding/csv"
+	"io"
+	"math/rand"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+)
+
+const DefaultInterval = 30 * time.Second
+
+var Rand = rand.New(rand.NewSource(time.Now().Unix()))
+
+type Backend interface {
+	Refresh() (map[string]Flag, error)
+}
+
+type fileBackend struct {
+	filename string
+}
+
+func (b fileBackend) Refresh() (map[string]Flag, error) {
+	f, err := os.Open(b.filename)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	return parseFlagsCSV(f)
+}
+
+type Flag struct {
+	Name string
+	Rate float64
```

Comment:
Agreed. I left it out because I didn't have a good use case for it in mind (e.g. we don't do per-merchant stuff in Go that often), but we can include later on.